src/otx/algorithms/segmentation/adapters/mmseg/task.py

```diff
@@ -162,13 +124,39 @@ def configure(
         recipe_cfg.resume = self._resume
 
         if self._train_type == TrainType.Incremental:
-            configurer = IncrSegmentationConfigurer("segmentation", training)
+            configurer = IncrSegmentationConfigurer(
+                "segmentation",
+                training,
+                export,
+                self.override_configs,
+                self.on_hook_initialized,
+                self._time_monitor,
+                self._learning_curves,
+            )
```

Comment:
Just nitpick: why don't we use kwarg names in case of many argument? At least in product code, not in test code. I cannot see which is which.

Reply:
I'll treat this in another following PR